### ISSUE TITLE
Receiving an AttributeError indicating that the API is returning XML data

### ISSUE THREAD

**Issue by kburchfiel** [mortada/fredapi#50]
After installing the latest version of fredapi and running the following code (from the library homepage):

```
from fredapi import Fred
fred = Fred(api_key=fred_key)
data = fred.get_series('SP500')

```
I received the following error message:

`[AttributeError: 'xml.etree.ElementTree.Element' object has no attribute 'getchildren']()`

It seems as though the API is retrieving XML data. How can I resolve this?

**Comment by kburchfiel** [comment]
Update: I installed the latest version available on conda-forge (0.42), but this is less recent than the version available through pip (0.50). So the error may be related to conda-forge's outdated fredapi version.

**Comment by mortada** [comment]
yes this issue should have been solved since version 0.4.3, similar discussion here: https://github.com/mortada/fredapi/issues/41
